backend/src/packages/users/user.service.ts

```diff
@@ -30,16 +42,31 @@ class UserService implements IService {
   public async create(
     payload: UserSignUpRequestDto,
   ): Promise<UserSignUpResponseDto> {
+    const saltRounds = 10;
+    const salt = await bcrypt.genSalt(saltRounds);
+    const hashedPassword = await bcrypt.hash(payload.password, salt);
```

Comment:
Create separate Encrypt package and use it here + add password salt rounds to config.ENCRYPTION
```suggestion
    const passwordSalt = await this.encrypt.generateSalt(
      this.config.ENCRYPTION.USER_PASSWORD_SALT_ROUNDS,
    );
    const hashedPassword = await this.encrypt.encrypt(
      payload.password,
      passwordSalt,
    );
```